packages/node/src/methods/app-instance/get-free-balance/controller.ts

```diff
@@ -25,16 +26,15 @@ export default class GetFreeBalanceController extends NodeController {
 
     const stateChannel = await store.getStateChannel(multisigAddress);
 
-    const {
-      alice,
-      bob,
-      aliceBalance,
-      bobBalance
-    } = stateChannel.getFreeBalanceFor(AssetType.ETH)
+    const appState = stateChannel.getFreeBalanceFor(AssetType.ETH)
       .state as ETHBucketAppState;
-    return {
-      [alice]: aliceBalance,
-      [bob]: bobBalance
-    };
+
+    const ret: Node.GetFreeBalanceStateResult = {};
+
+    for (const { amount, to } of appState) {
+      ret[to] = bigNumberify(amount._hex);
```

Comment:
Do you need to specify `._hex`? I think the function works correctly without doing that.

Reply:
it does not:

![image](https://user-images.githubusercontent.com/130362/57468511-938eba00-7252-11e9-9c23-e60240daf0f2.png)